conanfile.py

```diff
@@ -534,9 +547,10 @@ def _getenvpath(var):
 
         value = _getenvpath('CXX')
         if value:
-            args += ['QMAKE_CXX="' + value + '"',
-                     'QMAKE_LINK="' + value + '"',
-                     'QMAKE_LINK_SHLIB="' + value + '"']
+            args += ['QMAKE_CXX="' + value + '"']
```

Comment:
Is this change still needed with the `-nostdlib++` patch below ?

Reply:
You're right. That's a left over.